sass/custom/lewisham-and-greenwich-nhs-trust.scss

```diff
@@ -8,12 +8,12 @@ $govuk-font-family: -apple-system,BlinkMacSystemFont,helvetica neue,helvetica,ub
 $custom-banner-colour: #005EB8;
 $custom-banner-border-colour: #ffffff;
 $custom-text-colour: #ffffff;
-$logo-image-height: 2em;
+$logo-image-width: 215px;
```

Comment:
Maybe I'm not the best person to review this, but why did you change `logo-image-height` to `logo-image-width`? See this [commit](https://github.com/alphagov/pay-js-commons/commit/1700ebb936e38b7a09348c563e351dec2019d6e0) that right aligns the logo too.

Reply:
Our internal documentation advises us to do this.